Docker/validate.py

```diff
@@ -46,13 +50,13 @@ def main(submission, entity_type, results):
             invalid_reasons = [
```

Comment:
So I just read in a file that has duplicated nodes and it returned this to me:

> NewickReaderDuplicateTaxonError: Error parsing data source 'tree.nw' on line 1 at column 27: Multiple occurrences of the same taxa on trees are not supported: trees with duplicate node labels can only be processed if the labels are not parsed as operational taxonomic unit concepts but instead as simply node labels by specifying 'suppress_internal_node_taxa=True, suppress_leaf_node_taxa=True'. Duplicate taxon labels:  2

I think that function does return some useful information.  Might be helpful to return the error message or at least the error type: `NewickReaderDuplicateTaxonError`

Reply:
```
try:
    gs_tree = dendropy.Tree.get(file=open(gs_file, 'r'),
                            schema="newick",
                                tree_offset=0)
except Exception as e:
    error = e

print(type(error))
print(error.message)
print(str(error))
```